src/index.html

```diff
@@ -6,17 +6,54 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+          class="photo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="Apple"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li><a href="Samsung">SAMSUNG</a></li>
+          <li><a href="Smartphones">SMARTPHONES</a></li>
```

Comment:
Fix code style everywhere
```suggestion
          <li>
            <a href="Smartphones">SMARTPHONES</a>
          </li>
```